doc/testing-your-application.rst

```diff
@@ -4,22 +4,30 @@ Testing Your Application
 ========================
 
 This chapter describes how to test your application against your reverse proxy.
-
-The FOSHttpCache library provides base test case classes to help you write
-functional tests. This is helpful to test the way your application sets caching
-headers and invalidates cached content.
-
-By having your test classes extend one of the test case classes, you get:
-
-* independent tests: all previously cached content is removed in the tests
-  ``setUp`` method. The way this is done depends on which reverse proxy you use;
-* an instance of this library’s client that is configured to talk to your
-  reverse proxy server. See reverse proxy specific sections for details;
-* convenience methods for executing HTTP requests to your application:
-  ``$this->getHttpAdapter()`` and ``$this->getResponse()``;
-* custom assertions ``assertHit`` and ``assertMiss`` for validating a cache
+By running your tests against a live instance of your reverse proxy, you can
+validate the caching headers that your application sets, and the invalidation
+rules that it defines.
+
+The FOSHttpCache library provides traits and base test classes to help you write
+functional tests. Using the traits, you can extend your own (or your
```

Comment:
okay. then we should move those tests that are integration tests. imho:

* Unit test: tests a single method of a class
* Functional test: Tests the interaction of the application, e.g.
WebTestCase
* Integration test: Tests the interaction of several different systems,
e.g. a PHP application and Varnish